docs/visualization/normalization.rst

```diff
@@ -35,6 +35,44 @@ stretching is described in the `Stretching`_ section.
 Intervals and Normalization
 ===========================
 
+The quick way
+-------------
+
+We provide a convenience
+:func:`~astropy.visualization.mpl_normalize.simple_norm` function that can be
+useful for quick interactive analysis.:
```

Comment:
Nitpick:
```suggestion
useful for quick interactive analysis:
```

Reply:
Is no-period our standard?  I couldn't see the docs rendered for reasons I still don't understand.... will the period render badly?

Reply:
Did you try build the docs locally using `python setup.py build_docs`? 😉

Reply:
yes.  The pngs & svgs built, but no html?
```
cyg ~/repos/astropy vis_docs_revise$ ls docs/_build/html/visualization/
histogram-1.pdf                 matplotlib_integration-1.png    matplotlib_integration-5.png    normalization-3.py              rgb-1.svg
histogram-1.png                 matplotlib_integration-1.py     matplotlib_integration-5.py     normalization-3.svg             rgb-2.pdf
histogram-1.py                  matplotlib_integration-1.svg    matplotlib_integration-5.svg    normalization-4.pdf             rgb-2.png
histogram-1.svg                 matplotlib_integration-2.pdf    normalization-1.pdf             normalization-4.png             rgb-2.py
histogram-2.pdf                 matplotlib_integration-2.png    normalization-1.png             normalization-4.py              rgb-2.svg
histogram-2.png                 matplotlib_integration-2.py     normalization-1.py              normalization-4.svg             rgb-3.pdf
histogram-2.py                  matplotlib_integration-2.svg    normalization-1.svg             normalization-5.pdf             rgb-3.png
histogram-2.svg                 matplotlib_integration-3.pdf    normalization-2.pdf             normalization-5.png             rgb-3.py
histogram-3.pdf                 matplotlib_integration-3.png    normalization-2.png             normalization-5.py              rgb-3.svg
histogram-3.png                 matplotlib_integration-3.py     normalization-2.py              normalization-5.svg             wcsaxes
histogram-3.py                  matplotlib_integration-3.svg    normalization-2.svg             rgb-1.pdf
histogram-3.svg                 matplotlib_integration-4.py     normalization-3.pdf             rgb-1.png
matplotlib_integration-1.pdf    matplotlib_integration-5.pdf    normalization-3.png             rgb-1.py
```

Reply:
Eh... Do you see the same warnings that CI reported?

Reply:
Nope.  I see completely different warnings.  CI shows: 
```
"Warning, treated as error:
/home/circleci/project/docs/coordinates/skycoord.rst:881:py:obj reference target not found: numpy.random.multivariate_normal"
```
(wtf?)

locally, I see:
```
Warning, treated as error:
failed to import BaseDifferential.recommended_units
```
(wtf?)^2

Reply:
Hmm... `skycoord` one is fixed on `master`, so a rebase would get rid of it. As for the `BaseDifferential`, I am not sure...

Reply:
But back to the original point, if you use a colon, there is no reason to add a period before it?